tests/Element/TagBlockTest.php

```
@@ -192,6 +194,42 @@ public function testRenderWithBeginEnd(): void
+
+            protected function getTag(): Block
+            {
+                return Block::DIV;
+            }
+
+            protected function run(): string
+            {
+                if ($this->isBeginExecuted()) {
+                    $this->runBegin();
+
+                    return Html::end($this->getTag());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Html' in method 'testRenderWithBeginEndFromSubclassLifecycleHelpers'. (undefined)

(StaticAccess)